packages/tools/examples/contourRenderingConfiguration/index.ts

```diff
@@ -341,21 +269,19 @@ async function run() {
     ]
   );
 
-  const segRepresentations2 = await segmentation.addSegmentationRepresentations(
-    toolGroupId3d,
-    [
-      {
-        segmentationId: `${segmentationId}`,
-        type: csToolsEnums.SegmentationRepresentations.Contour,
-      },
-    ]
-  );
-
   planarSegmentationRepresentationUID = segRepresentations1[0];
-  volumeSegmentationRepresentationUID = segRepresentations2[0];
 
   // Render the image
   renderingEngine.render();
+  segmentation.config.setToolGroupSpecificConfig(toolGroupId, {
+    renderInactiveSegmentations: true,
+    representations: {
+      CONTOUR: {
+        outlineWidthActive: 1,
+        renderFill: false,
+      },
```

Comment:
I think these should be default in contour rendering, can you move it internallly? so that we can remove the whole 

```
segmentation.config.setToolGroupSpecificConfig(toolGroupId, {
    renderInactiveSegmentations: true,
    representations: {
      CONTOUR: {
        outlineWidthActive: 1,
        renderFill: false,
      },
    },
  });
  ```